docs/user/ppl/functions/collection.rst

```diff
@@ -216,19 +216,74 @@ Return type: STRING
 
 Example::
 
-    PPL> source=people | eval result = mvjoin(array('a', 'b', 'c'), ',') | fields result | head 1
+    os> source=people | eval result = mvjoin(array('a', 'b', 'c'), ',') | fields result | head 1
+    fetched rows / total rows = 1/1
+    +--------+
+    | result |
+    |--------|
+    | a,b,c  |
+    +--------+
+
+    os> source=accounts | eval names_array = array(firstname, lastname) | eval result = mvjoin(names_array, ', ') | fields result | head 1
+    fetched rows / total rows = 1/1
+    +-------------+
+    | result      |
+    |-------------|
+    | Amber, Duke |
+    +-------------+
+
+MVAPPEND
+--------
+
+Description
+>>>>>>>>>>>
+
+Version: 3.4.0
+
+Usage: mvappend(value1, value2, value3...) appends all elements from arguments to create an array. Flattens array arguments and collects all individual elements. Always returns an array or null for consistent type behavior.
```

Comment:
In example, add following use cases
1. Handle dupldate values. mvappend(1, 1)
2. Handle null values. mvappend(1, null)
3. Nested use case, mvapend(1, mvappend(2, 3))